taxi/tests/test_models.py

```diff
@@ -0,0 +1,57 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.models import Manufacturer, Driver, Car
+
+
+class ModelTests(TestCase):
+    def test_manufacturer_str(self):
+        manufacturer = Manufacturer.objects.create(name="test",
+                                                   country="test")
+        self.assertEqual(str(manufacturer),
+                         f"{manufacturer.name} {manufacturer.country}")
```

Comment:
The string representation of the manufacturer should match the format specified in the model's `__str__` method. Ensure that the `__str__` method in the Manufacturer model returns the expected format.